### PR TITLE
Update the comment on the message_type to cover what it should be.

### PR DESCRIPTION
**Pull request by thomasvl**

### CODE REVIEW

File: conformance/conformance.proto [hunk 1]
```diff
@@ -78,7 +78,9 @@ message ConformanceRequest {
   // Which format should the testee serialize its message to?
   WireFormat requested_output_format = 3;
 
-  // should be set to either "proto2" or "proto3"
+  // The full name for the test message to use; either
```
Comment (haberman) [inline conformance/conformance.proto]: Maybe better to say: "for the moment, either:"

We might have reason to add others in the future. Implementations that can lookup messages by name should just do that.

File: csharp/src/Google.Protobuf.Conformance/Conformance.cs [hunk 2]
```diff
@@ -142,7 +142,9 @@ public sealed partial class ConformanceRequest : pb::IMessage<ConformanceRequest
     public const int MessageTypeFieldNumber = 4;
     private string messageType_ = "";
     /// <summary>
-    /// should be set to either "proto2" or "proto3"
+    /// The full name for the test message to use; either
```
Comment (haberman) [inline csharp/src/Google.Protobuf.Conformance/Conformance.cs]: ditto

File: conformance/conformance.proto (hunk 1 above)
Comment (thomasvl) [inline conformance/conformance.proto]: Commit amended.

File: csharp/src/Google.Protobuf.Conformance/Conformance.cs (hunk 2 above)
Comment (thomasvl) [inline csharp/src/Google.Protobuf.Conformance/Conformance.cs]: regenerated to get it pulled here also.

**Pull request merged**